src/platform/plugins/shared/controls/public/control_group/components/floating_actions.tsx

```diff
@@ -138,3 +142,32 @@ export const FloatingActions: FC<FloatingActionsProps> = ({
     </div>
   );
 };
+
+const useStyles = () => {
+  const { euiTheme } = useEuiTheme();
+  const styles = useMemo(() => {
+    return {
+      wrapperStyles: css({
+        position: 'relative',
+        '&:hover, &:focus-within': {
+          '.presentationUtil__floatingActions': {
+            opacity: 1,
+            visibility: 'visible',
+            transition: `visibility ${euiTheme.animation.fast}, opacity ${euiTheme.animation.fast}`,
+          },
+        },
+      }),
+      floatingActionsStyles: css({
+        opacity: 0,
+        visibility: 'hidden',
+        // slower transition on hover leave in case the user accidentally stops hover
+        transition: `visibility ${euiTheme.animation.slow}, opacity ${euiTheme.animation.slow}`,
```

Comment:
Unless I'm mistaken, I don't believe `visibility` can be transitioned in CSS. I think you can safely remove that part of the transition altogether.

```suggestion
        transition: `opacity ${euiTheme.animation.slow}`,
```